jobrunner/run.py

```diff
@@ -550,20 +533,42 @@ def set_code(job, code, message, error=None, results=None, timestamp_ns=None, **
         timestamp_s = int(t)
         timestamp_ns = int(t * 1e9)
     else:
-        timestamp_s = int(timestamp_ns // 1e9)
-
-    if job.status_code_updated_at > timestamp_ns:
-        # we somehow have a negative duration, which honeycomb does funny things with.
-        # This can happen in tests, where things are fast, but we've seen it in production too.
-        log.warning(
-            f"negative state duration, clamping to 1ms ({job.status_code_updated_at} > {timestamp_ns})"
-        )
-        timestamp_ns = job.status_code_updated_at + 1e6  # set duration to 1ms
-        timestamp_s = int(timestamp_ns // 1e9)
+        timestamp_s = int(timestamp_ns / 1e9)
```

Comment:
double-checking this should be `/` and not `//` 
```suggestion
        timestamp_s = int(timestamp_ns // 1e9)
```

Reply:
yep, agreed